docs/source/usage/how_to_run.rst

```diff
@@ -3,87 +3,89 @@
 Run ImpactX
 ===========
 
-In order to run a new simulation:
 
-#. create a **new directory**, where the simulation will be run
-#. make sure the ImpactX **executable** is either copied into this directory or in your ``PATH`` `environment variable <https://en.wikipedia.org/wiki/PATH_(variable)>`__
-#. add an **inputs file** and on :ref:`HPC systems <install-hpc>` a **submission script** to the directory
-#. run
+When to run: Physics Models
+---------------------------
 
-1. Run Directory
-----------------
+ImpactX most powerful and central physics model is its symplectic particle tracker with collective effects (space charge, CSR, wakefields, etc.).
```

Comment:
```suggestion
ImpactX can be run using any of three distinct tracking modes.  ImpactX most powerful and central physics model is its symplectic particle tracker with collective effects (space charge, CSR, wakefields, etc.).
```

Reply:
I would not call these "physics models".  They really are modes that differ due to what is being tracked--for example, for linear beamline elements (default) the physical model is the same in all three cases.

Reply:
Agreed, thx!